src/constants/standards.ts

```diff
@@ -3,4 +3,7 @@ export default {
   icpunks: 'ICPunks',
   departuresLabs: 'DepartureLabs',
   erc721: 'ERC721',
+  dip20:  'DIP20',
+  xtc: 'XTC',
+  wicp: 'WICP'
```

Comment:
I think asset and nft standards should be separate for better clarity. I don't mind EXT being repeated either.